e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -387,9 +387,9 @@ describe("scenarios > dashboard > temporal unit parameters", () => {
         .should("contain.text", "Started from")
         .should("contain.text", multiBreakoutQuestionDetails.name);
       queryBuilderMain().within(() => {
-        cy.findByText("Product → Created At: Week").should("be.visible");
-        cy.findByText("2022").should("be.visible");
-        cy.findByText("2023").should("be.visible");
+        cy.findByText("Created At: Year").should("be.visible");
```

Comment:
Before this PR: default pivot column is "Created At": [image](https://github.com/user-attachments/assets/025ff332-a337-46bf-b563-0bc78b16b846)

After this PR: default pivot column is "Product -> Created At": [image](https://github.com/user-attachments/assets/4c27cc81-961c-46fd-89b6-9bea062cd514)

Arguably it was correct/better before so I'd say this is a regression.

Reply:
Nope this is the expected behavior, and there was a bug ticket to fix it. But I didn't realize that it was an issue with viz settings